### ISSUE TITLE
Create struct for parsing result

### ISSUE THREAD

**Issue by lunchboxav** [lunchboxav/wayang-rs#2]
Right now, each individual parsing results are stored on its own separate vectors. It should be formalized and stored in its own struct. Something like

```
struct WygStory {
  title: &str
  scene: Vec<&str>
  events: Vec<&str>
  choices: Vec<Link>
}
```

Consequently, some parts of the code will need to reflect these changes as well.

**Comment by asadnq** [comment]
can i work on this issue?

**Comment by lunchboxav** [comment]
sure sure, it's open. I'm assigning it to you then :) good luck. If any questions arise, feel free to discuss it here.

**Comment by lunchboxav** [comment]
closed via #4
